proj/src/main/webapp/chat.js

```diff
@@ -434,14 +432,24 @@ function initRef(dbRefObject) {
     // note that when a comment is added it will display more than the limit, which
     // is intentional
     dbRefObject.off('child_added');
-    dbRefObject.orderByChild("timestamp").limitToLast(LIMIT + 1).on('child_added', snap => {
-        if (!firstChildKey) {
-            firstChildKey = snap.key;
-        } else {
-            messageDom = createMessageWithTemplate(snap.key, snap.val());
-            chat.appendChild(messageDom);
-        }
-    });
+    blockedRef = firebase.database().ref('/users/'+currentUID+'/blocked')
```

Comment:
I'm not sure if this line will be resolved by the other PR, or if it will cause merge conflicts, but can this `currentUiD` be found inside the function instead of pulled from the global variable.

Reply:
fixed for real this time. i did it once but i think a accidentally deleted the changed when switching branches -- oops